docs/modules/ROOT/pages/cops_metrics.adoc

```diff
@@ -446,7 +446,7 @@ end               # 6 points
 | 0.87
 |===
 
-Checks if the length a module exceeds some maximum value.
+Checks if the length of a module exceeds some maximum value.
```

Comment:
This docs/modules/ROOT/pages/cops_metrics.adoc is auto-generated file. Please edit lib/rubocop/cop/metrics/class_length.rb original file instead. So, edits to this adoc file is not required.

Reply:
Thank you for the clarification, I've made the change.